### PR TITLE
Adding/Changing Unit Descriptions

### PR DESCRIPTION
**Pull request by Gothyoba**
This PR aims to add unit descriptions to Wesnoth units, or replace unit descriptions which have been copied from other units. More units can be worked on, and changes can be made to these new descriptions. This is intended to make the game feel deeper, more complete and implement more lore around these units. The following units have been worked on in this PR:

Direwolf - This one had the same description as a wolf. I took some lines from the description for the Direwolf Rider.

Great Wolf - Like the Direwolf, this unit copied its description from the wolf.

Wyvern Rider - This one had no description.

You may suggest changes to these descriptions. While my Great Wolf description does not expand much on the game, I feel like not much can be done there. Adding this description could still make wolves feel more complete though. I am also expanding on Direwolf lore through the Direwolf Rider. I would personally consider this PR to be complete. Is this currently eligible for merging?

Note: I will not be working on Wose Sapling in this PR

### CODE REVIEW

**Review by TheShadowOfHassen — CHANGES_REQUESTED** [review]
So in the great wolves the sentence "A single one may be kill a soldier as easily as whole pack if found unprepared."

Isn't 100% clear. I understand where you're going. (A single wolf can kill a soldier) but I think it needs another editing pass.

**Comment by Gothyoba** [comment]
I scrapped that sentence completely in favour of something which I think sounds better, though it might show the, as more powerful than ithey reallyis:

“ They typically stand taller than a horse at the shoulder, and have the strength and appetite to match.”

A little more editing may be necessary.

Should something about the Ashland Desert be added to the Wyvern Rider description?

**Comment by nemaara** [comment]
I don't think I can add these descriptions in but if you really want descriptions for these units I can write them.

**Comment by nemaara** [comment]
The thing about unit descriptions and encyclopedia entries is that they should add some lore or other background info. If you just copy stuff from other descriptions or add something generic, it doesn't really add anything. The sapling description you have does add something but I don't think that's the direction we should be taking woses in. The other 3 descriptions really don't add much for me.

**Comment by Gothyoba** [comment]
On the Great Wolf and Driewolf:

While descriptions for these are unlikely to be capable of being good and expand the lore, having these might make the game feel more deep. Currently it just feels like a work in progress without a descirotion. Direwolve aren’t even wolves, yet they are reffered to in that way in their description. At least that should be changed to keep things consistent.

On the Wose Sapling:

i think I will give up on this, thoguh I will make an issue on the topic, as I think much could be made to expand wose lore here.

On the Wyvern Rider:

This unit just feels lacking. It is not used anything a description could at least make it feel like its actually worked on/useful for lore. It is cetainly lacking when it come to expanding the lore, and I will try to change that. However, I will take my time on this as I don’t have any particularly good ideas at the moment. I was thinking that the description could focus more on the taming of the wyvern, how it is kept and perhaps a few more details of the rider’s place in society, or what the Wyvern means to the Dunefolk (e.g. as a sacred creature of their culture). Wyvern Riders would then only rarely fight in battle.

Overall

This PR will continue as a draft, focusing largely on lore for the Wyvern Rider. Should I also make a post about Wyvern Rider on the Wensoth Forums? I feel like more people would comment there a beter concensus can be reached.

**Comment by nemaara** [comment]
Oh I agree with you on the feeling like a WIP that's why I'm willing to write the descriptions. But it's better to not add a description that doesn't really say anything, that's my point.

Point is if we add something we do have to put some thought into lore and what the unit really is, so it's not just something trivial.

**Comment by Gothyoba** [comment]
Idea for Direwolf deacription:

Perhaps this could foucs less on the Direwolf and more on how it is seen by other races (e.g. feared among Wesnothians, but considered sacred to orcs).

**Comment by nemaara** [comment]
It's a good idea, I would suggest moving it over to the Direwolf Rider description and then add the stuff pertaining just to the direwolf (i.e. taller than a horse and strength and appetite and whatnot) to the direwolf unit itself.

**Comment by Gothyoba** [comment]
Thanks fort the feedback! I have done something similar to that, though I think I might also add more the wolf/wolf rider descriptions. As for the Wyvern rider, I think that Wyverns might be a feared yet respected symbol of the dunefolk, and those who ride them may be considered honoured among their people for having done what few others have thought of: tame the Wyvern. An issue has been openned for theWose Sapling’s lack of any description: 

https://github.com/wesnoth/wesnoth/issues/7685

**Comment by Gothyoba** [comment]
Great Wolves probably don’t have much more lore to put that isn’t just normal wolf lore, and I don’t think there is a good way of making it exoand the lore or give background info and not feel like an unpolished work in progress. For this reason, I do not think I will change this description from the proposal I gave here. You may suggest if you believe these goals may actually be possible for great wolves.

**Comment by Gothyoba** [comment]
Upon inspecting my descriptions, I think that this can be eligible for merging, unless the Great Wolf description is not allowed.

File: data/core/units/goblins/Direwolf_Rider.cfg [hunk 1]
```diff
@@ -16,9 +16,9 @@
     {AMLA_DEFAULT}
     cost=44
     usage=scout
-    description= _ "‘Dire wolves’ differ from the common variety only in size and color. They typically stand taller than a horse at the shoulder, and have an appetite to match. Only a madman would willingly encounter them; but goblins, at great cost to themselves, have managed to tame and ride them.
-
-The claws of a wolf are not generally regarded as being the more dangerous parts, but on a beast this large, they are thicker and longer than iron nails. Their riders smear a poison on the front set, not unlike that used by the orcish assassins, making a swipe from these beasts’ paws into a most deadly stroke."
+    description= _ "While the danger of direwolves may make them a bad omen in Wesnoth, the orcs and goblins see things differently. They have learned to create strong bonds with Wolves. To them Direwolves are valued almost as high as themselves and they are considered a symbol of orcish perseverance.
```
Comment (knyghtmare) [inline data/core/units/goblins/Direwolf_Rider.cfg:19]: I can see goblins doing it but orcs? Not so sure..it sounds okay...I think.

File: data/core/units/goblins/Direwolf_Rider.cfg [hunk 2]
```diff
@@ -16,9 +16,9 @@
...
+    
+In fact, the fangs of a direwolf that are shown on the flag of the Whitefang Orcs. Occasionally, a goblin may be given the rare honor of riding a Direwolf into battle, where they may be deadly to those attacked by them. This is one of the few great honors exclusive to the goblins in their society, as the Orcs are to large to ride on a wolf's back, even on that of a Direwolf."
```
Comment (knyghtmare) [inline data/core/units/goblins/Direwolf_Rider.cfg:21]: "to" missing the extra "o"

File: data/core/units/monsters/Wolf_Dire.cfg [hunk 3]
```diff
@@ -16,7 +16,7 @@
     advances_to=null
     cost=33
     usage=scout
-    description=_ "Wolves are aggressive animals that hunt in packs. Although weak individually, a wolf pack can kill even the strongest man in minutes."
+    description=_ "Direwolves are stronger, darker, larger and more dangerous then wolves, making them feared among many travellers. Even their claws can be considered as deadly as a wolf's bite. They typically stand taller than a horse at their shoulder, and have an appetite to match."
```
Comment (knyghtmare) [inline data/core/units/monsters/Wolf_Dire.cfg:19]: your suggested description says they have claws secondary attack but they dont. The Direwolf Rider does, not the direwolf itself.

File: data/core/units/monsters/Wolf_Great.cfg [hunk 4]
```diff
@@ -16,7 +16,7 @@
     advances_to=null
     cost=21
     usage=scout
-    description=_ "Wolves are aggressive animals that hunt in packs. Although weak individually, a wolf pack can kill even the strongest man in minutes."
+    description=_ "The greatest of wolves may almost possess the strength of a direwolf, and they should never be underestimated, for those who have have suffered terrible fates."
```
Comment (knyghtmare) [inline data/core/units/monsters/Wolf_Great.cfg:19]: I can see why you used 2x "have" words here. maybe a comma in between?

File: data/core/units/dunefolk/Wyvern_Rider.cfg [hunk 5]
```diff
@@ -14,7 +14,9 @@
     {AMLA_DEFAULT}
     cost=102
     usage=scout
-    description=""
+    description=_ "To the Dunefolk, the Wyvern is an image of untamable fear that may ravage its enemies, yet also a creature to be seen with great respect. The Wyvern is even the central piece of the united Dunefolk banner. The oDunefolk who manage to tame the Wyvern are few and far between, greatly revered among their kind as great heroes, even being passed on into legend. These riders devote their lives to the Wyvern, keeing whole packs of Wyverns in the mountains and hills near the desert. Yet even the riders may form strong bonds with the Wyverns, sometimes almost become a part of the pack.
```
[the remark this answers is not in the document]
Comment (Gothyoba) [reply data/core/units/dunefolk/Wyvern_Rider.cfg]: Corrected

[the remark this answers is not in the document]
Comment (Gothyoba) [reply data/core/units/dunefolk/Wyvern_Rider.cfg]: Thanks

File: data/core/units/dunefolk/Wyvern_Rider.cfg [hunk 6]
```diff
@@ -14,7 +14,9 @@
...
+
+Some Dunefolk leaders may hire these people for their courts to show their great power and wealth. These royal wyverns are rarely sent to battle, yet when they are it is usually to show one's power and incit fear in the enemy, demosntrating that your forces are strong enough to have tamed the Wyvern."
```
[the remark this answers is not in the document]
Comment (Gothyoba) [reply data/core/units/dunefolk/Wyvern_Rider.cfg]: Thanks also corrected

File: data/core/units/monsters/Wolf_Dire.cfg (hunk 3 above)
Comment (Gothyoba) [reply data/core/units/monsters/Wolf_Dire.cfg]: I removed this line.

File: data/core/units/goblins/Direwolf_Rider.cfg (hunk 2 above)
Comment (Gothyoba) [reply data/core/units/goblins/Direwolf_Rider.cfg:21]: I don't think I even use that word here. Where is this?

File: data/core/units/monsters/Wolf_Great.cfg (hunk 4 above)
Comment (Gothyoba) [reply data/core/units/monsters/Wolf_Great.cfg]: I just removed it. I sounds better. Thank you for pointing this out.

File: data/core/units/goblins/Direwolf_Rider.cfg (hunk 2 above)
Comment (knyghtmare) [reply data/core/units/goblins/Direwolf_Rider.cfg:21]: > I don't think I even use that word here. Where is this?

"the Orcs are `to` large to ride on"

Comment (Gothyoba) [reply data/core/units/goblins/Direwolf_Rider.cfg]: Sorry I didn't notice that it has been corrected thanks for pointing it out

**Comment by knyghtmare** [comment]
I reviewed your corrections but I just left one unresolved since I am not sure about the dire-wolves being held in high regard by orcs topic.

**Comment by Gothyoba** [comment]
Orc sand goblins are in thes ame society, so they should probably both value direwolves. Orcs may not form such strong bonds with them, as they do not have to ride them, but it could still be a symbol of pride. I think it would be strange if nly goblins valued the direwolves.

**Comment by Gothyoba** [comment]
So I wa# thinking about this and maybe the direwolves are specifically revered among the the goblin’s social circles, as this would probably a rather segreagted society. Perhaps instead of symbolizing orcish perseverance, it symbolizes the greatness a goblin can have despite not being born a wolf. In response, I might change the descriptions to allow for this new lore. I would appereciate any feedback on this.

**Comment by Gothyoba** [comment]
A few changes have been made and this should be ready for review now.

**Comment by Gothyoba** [comment]
This is now a draft again, as I think the Wyvern Rider should be discussed further. Here is a forum post I made on the topic:

https://forums.wesnoth.org/viewtopic.php?t=56975

**Comment by Gothyoba** [comment]
It has been five days since I openned the forum post on the Wyvern Rider description. One person answered and the problems they pointed out have been adressed. With nothing else to check, and this being in line with knyghtmare’s ideas (I think), this should be ready for review, and oerhaps, eventually, merging.

**Comment by nemaara** [comment]
I think the concepts in the Direwolf Rider description is good. For the Wyvern Rider, I think it's better if I spend some time coming up with the concept, since I wrote the description for the rest of the faction. Alternatively, you can try again with a different concept. I just don't think the current one works, the rest of the faction's vibe is about science and reason so I don't really want to have one where they're "revering" Wyverns + their riders. I also don't really like the Great Wolf description so let's table that one.

I'll help you one the wording later once this is not a draft.

**Comment by Gothyoba** [comment]
Thank you for your feedback.

On the Wyvern Rider:

Personally, I think that with some modification, this description could work. However, perhaps oher ideas could still be better in their place. I recomend reaching concensus in this forum post and removing the Wyvern from this PR: https://forums.wesnoth.org/viewtopic.php?t=56975.

On the Great Wolf:

I think I will give this one another revision. Looking at the description Inproposed, it not only lacks any expansion on lore, but it does not even sound akin to other descriptions in Wesnoth. I think I might experiment with making Great Wolves pack leaders, and talking about their role or strength in wolf society, ot maybe how others see them.

On the Direwolf and Driewolf Rider:

I dont think I will make many major chages here.

Overall:

This PR will likely as a draft, and at least temporarily remving the Wyvern Rider.

**Comment by Hejnewar** [comment]
Just small comment on the wyvern riders, great horses is a big underestimation. I would go as far as to say that they are probably only used by Royal family or very powerful individuals. 

I should probably write or at least try to write wild wyvern description and maybe buff them.

**Comment by sevu** [comment]
@Gothyoba @nemaara  If this should be part of the next release, it needs to be merged within the next weeks.

**Comment by nemaara** [comment]
Okay I added some descriptions in 2375d5fde33. Make a new PR if you want to try the other ones besides the wolves.

**Comment by CelticMinstrel** [comment]
That commit doesn't have any unit description changes.

**Comment by soliton-** [comment]
The commit is ec5bb96.

**Pull request closed without merging**